build/ci/backend/docker/install_mu.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-S3_URL=https://s3.us-east-2.amazonaws.com/convertor.musescore.org
+S3_URL=https://s3.amazonaws.com/convertor.musescore.org
```

Comment:
and here

Reply:
I'll rename these later, it's more difficult with it